code/controllers/subsystems/trade.dm

```diff
@@ -274,35 +248,79 @@ SUBSYSTEM_DEF(trade)
 	var/count_of_all = collect_counts_from(shopList)
 	var/price_for_all = collect_price_for_list(shopList, station)
 	if(isnum(count_of_all) && count_of_all > 1)
-		price_for_all += station.commision
 		C = senderBeacon.drop(/obj/structure/closet/crate)
 	if(price_for_all && get_account_credits(account) < price_for_all)
 		return
 
 	for(var/categoryName in shopList)
 		var/list/shoplist_category = shopList[categoryName]
-		var/list/assortiment_category = station.assortiment[categoryName]
-		if(length(shoplist_category) && length(assortiment_category))
+		var/list/inventory_category = station.inventory[categoryName]
+		if(length(shoplist_category) && length(inventory_category))
 			for(var/pathOfGood in shoplist_category)
 				var/count_of_good = shoplist_category[pathOfGood] //in shoplist
-				var/index_of_good = assortiment_category.Find(pathOfGood) //in assortiment
+				var/index_of_good = inventory_category.Find(pathOfGood) //in inventory
 				for(var/i in 1 to count_of_good)
 					istype(C) ? new pathOfGood(C) : senderBeacon.drop(pathOfGood)
 				if(isnum(index_of_good))
 					station.set_good_amount(categoryName, index_of_good, max(0, station.get_good_amount(categoryName, index_of_good) - count_of_good))
 	station.add_to_wealth(price_for_all)	// can only buy from one station at a time
 	charge_to_account(account.account_number, account.get_name(), "Purchase", station.name, price_for_all)
 
-/datum/controller/subsystem/trade/proc/sell_thing(obj/machinery/trade_beacon/sending/senderBeacon, datum/money_account/account, atom/movable/thing, datum/trade_station/station)
-	if(QDELETED(senderBeacon) || !istype(senderBeacon) || !account || !istype(thing) || !istype(station))
+/datum/controller/subsystem/trade/proc/export(obj/machinery/trade_beacon/sending/senderBeacon)
+	if(QDELETED(senderBeacon) || !istype(senderBeacon))
 		return
 
-	var/cost = get_sell_price(thing, station)
+	var/sold_str = ""
+	var/cost = 0
 
-	if(account)
-		qdel(thing)
-		senderBeacon.activate()
-		var/datum/money_account/A = account
-		var/datum/transaction/T = new(cost, account.get_name(), "Sold item", station.name)
-		T.apply_to(A)
-		station.add_to_wealth(cost)
+	for(var/atom/movable/AM in senderBeacon.get_objects())
+		if(isliving(AM))
+			// Add very light burn damage plus moderate hal damage for living things in the beacon area
+			var/mob/living/L = AM
+			L.apply_damages(0,5,0,0,0,5)
+			// TODO: Small chance to export players to deepmaint hive import beacon
```

Comment:
Wut?

Reply:
Anyone standing in the sending beacon area will get zapped. The deepmaint thing was an idea I had that tied into a separate project I've been working on.